<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGISReposDialog</class>
 <widget class="QDialog" name="QGISReposDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QGIS repositories</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pbAddRepo">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbEditRepo">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbDelRepo">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
